stubs/PyMySQL/pymysql/charset.pyi

```diff
@@ -1,15 +1,22 @@
-from typing import Any
+from collections.abc import Callable
+from typing import Final
 
-MBLENGTH: Any
+MBLENGTH: Final[dict[int, int]]
 
 class Charset:
     is_default: bool
-    def __init__(self, id, name, collation, is_default: bool = False) -> None: ...
+    def __init__(self, id: str, name: str, collation, is_default: bool = False) -> None: ...
```

Comment:
What makes you think `id` is a str? The `is_binary` method compares it with an int.